example_queries/return.cc

```diff
@@ -168,36 +168,36 @@ void BidiContext::onResponseHeadersInbound() {
   // From rust code, we'll pass down, a vector of vector of strings.
   // and generate following snippet for each of the inner vector.
   {
-  int64_t value;
+  std::string value;
   if (getValue({
-      "response","total_size",
```

Comment:
Was this substitution automatically generated?  Or did you manually put it in?  Also have these been tested on minikube?

Reply:
These files are reference files produced by the latest version of the compiler. Minikube tests worked still.

Reply:
Okay, sounds good.  They looked correct, I was just surprised it had to be changed.